app/views/videos/index.html.haml

```diff
@@ -1,9 +1,9 @@
 %article.video_category
   %header
     - @categories.each do |category|
-      - category.videos.each do |video|
-        %h3= link_to(video.category.name, video.category) # same as: link_to("#{video.category.name}", category_path(video.category))
-        .videos.row
-          .video.span2
+      %h3= link_to(category.name, category) # same as: link_to("#{video.category.name}", category_path(video.category))
```

Comment:
Yes its same as your comment, but I don't like to see this part in code review. Comments like this do not belong to your code base, what other people can read.